zbus/tests/e2e.rs

```diff
@@ -257,7 +257,7 @@ impl MyIface {
 
     #[instrument]
     #[zbus(property)]
-    fn count(&self) -> u32 {
+    fn count(&self, #[zbus(header)] hea: Option<Header<'_>>) -> u32 {
```

Comment:
Kudos for adding a test case.

```suggestion
    fn count(&self, #[zbus(header)] header: Option<Header<'_>>) -> u32 {
```

Also let's do this in another (perhaps new) property and actually use the header. (you don't get a warning from the compiler about unused param? :thinking:)

Reply:
No unused parameter warning, no. Maybe Rust is more permissive with `Option`s? Not sure.